app/src/main/res/layout/ongoing_topic_item.xml

```diff
@@ -79,16 +78,15 @@
           style="@style/TextViewStart"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginStart="8dp"
           android:layout_marginTop="4dp"
-          android:layout_marginEnd="8dp"
```

Comment:
These margins are not required anymore because correct constraints have been added at bottom.